spec/features/course/experience_points_record_management_spec.rb

```diff
@@ -61,6 +87,13 @@
         expect(page).to have_content_tag_for(record)
         expect(page).to have_content_tag_for(manual_record)
         expect(page).not_to have_content_tag_for(inactive_record)
+
+        # can view but cannot edit
```

Comment:
Just to check, these are the form field classes right? Maybe can specify in the comments: `Can view experience points attributes but cannot edit the experience points record`.